app/jobs/get_jobs_job.rb

```diff
@@ -0,0 +1,9 @@
+class GetJobsJob < ApplicationJob
+  queue_as :default
+
+  def perform
+    SitesUtil.params.each do |params|
+      SearchRemoteJobsService.new(params).execute
+    end
```

Comment:
```suggestion
    SitesUtil.params.each { |params| SearchRemoteJobsService.new(params).execute }
```
Acho que da pra fazer numa mesma linha já que é um item só, a não ser que o rubocop fale que ta muito grande a linha. (Acho 80 caracteres bem pouco 😓 )

Reply:
fica muita linha para ler, só se far ,numa linha quando os nomes são pequenos e de fácil entendimento, e sim passa de 80